packages/integration-karma/test/shadow-dom/event-in-shadow-tree/event-post-dispatch.spec.js

```diff
@@ -27,7 +27,7 @@ describe('Event properties post dispatch on node in a shadow tree', () => {
         nodes = createShadowTree(document.body);
     });
 
-    // TODO [#1129]: Invoking dispatchEvent on nodes rendered by the template doesn't respect retargetting
```

Comment:
Instead of keeping those tests disabled, let's enable those tests and have 2 different assertions for native and synthetic shadow. I think it's important that those tests run even if we expect different results. This will ensure that we don't introduce regression in the future with the synthetic shadow for example.

Reply:
There is something fishy about this test file. The tests in this suite all pass fine. Enabling tests is the second suite will cause some in this suite to start failing. Seems like some issue with clean up. Have to investigate more. Will do a separate PR for this one. Reverting this file.

Reply:
@pmdartus Made a comment on issue #1129 https://github.com/salesforce/lwc/issues/1129#issuecomment-574889703
Events are patched only before a listener is invoked. If the event is accessed outside of a listener, the event appears unpatched. 

I have enabled the tests in native-shadow mode. For synthetic-shadow, there is no functionality to assert. IMO, it would be incorrect to assert the wrong behavior, so these tests don't have to be run.